src/adaptors/wise-lending-v2/index.js

```diff
@@ -126,9 +125,11 @@ const getTokenData = async (chain, token, addresses, contract) => {
       project: projectSlug,
       symbol: token,
       poolMeta: usdPrice.symbol,
-      tvlUsd,
+      tvlUsd: totalSupplyUsd,
```

Comment:
for lending pools we use tvlUsd = available liquidity, so totalSupplyUsd - totalBorrowUsd

Reply:
@slasher125 would that pool be picked up if we have $20k in the pool but $15k is borrowed, hence requirement of tvlUSD > $10k suppose we have $90k supplied and $81k borrowed we want our pools to be listed on this page!
https://defillama.com/protocol/wise-lending-v2#yields

Reply:
@slasher125 I guess you can disregard this PR and just review this one instead: https://github.com/DefiLlama/yield-server/pull/1450

Reply:
@slasher125 I still have a question if those pools that have more than 10k in total supplied would be displayed if users borrowed funds from them utilizing more than 90% of the pool, hence we can have 20k in the pool but 15k is borrowed out... we want these pools to still be visible.